sram.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+"""Functions for communicating with SRAM and some utilities."""
+
+__copyright__ = 'Copyright (c) 2019-2022, Utrecht University'
+__license__ = 'GPLv3, see LICENSE'
+
+import requests
+
+import mail
+from util import *
+
+
+def sram_post_collaboration(ctx, payload):
+    """Create SRAM Collaborative Organisation Identifier."""
+
+    url = "{}/api/collaborations/v1".format(config.sram_rest_api_url)
+    headers = {'Content-Type': 'application/json', 'charset': 'UTF-8', 'Authorization': 'bearer ' + config.sram_api_key}
+
+    response = requests.post(url, json=payload, headers=headers, timeout=30)
```

Comment:
Add new configuration option for TLS verification:
`verify=config.sram_tls_verify`